collectors/0/samza_staging.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+import json
+import re
+import sys
+
+from kafka import KafkaConsumer
+
+from collectors.lib import utils
+
+ENV = 'staging'
+
+def main(argv):
+
+    utils.drop_privileges()
+
+    # Pulls metrics from the "samza_metrics" Kafka topic
+    consumer = KafkaConsumer('samza_metrics',
+                           group_id='samza_metrics',
+                           bootstrap_servers=['kafkastaging-1b-east-fd8d3028-resonantdromedary.us-east-1.optimizely:9092',
+                                              'kafkastaging-1e-east-0a3cafa9-enchantedrhinoceros.us-east-1.optimizely'])
+
+    for message in consumer:
+
+        metric_json = json.loads(message.value)
+        metric_header = metric_json['header']
+        ts = int(metric_header['time'] / 1000)
+
+        metric_header['env'] = ENV
+        # Only 5 tags allowed.
+        metric_header['metric-source'] = "%s.%s.%s.%s" % \
```

Comment:
Per discussion, we will include the following tags for now:
- job-name
- host
- container-name

Will remove `env` tag and make job-name something like `samza-nogoalids-staging`.